index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tibute Page</title>
+    <link rel="stylesheet" href="./css/tribute_style.css">
+</head>
+
+<body>
+    <div class="main" id="main">
+        <div class="main_content">
+            <div class="main_header">
+                <h1 id="title" style="font-size: 48px;">Dr. Norman Borlaug</h1>
+                <p style="font-size: 18px;">The man who saved a billion lives</p>
+            </div>
+            <div class="image" id="img-div">
+
+                <img id="image" src="./img/tribute-page-main-image.jpg">
+                <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</div>
+            </div>
+            <div id="tribute-info">
+                <p style="font-size: 20px;padding-top: 30px;"><b>Here's a time line of Dr. Borlaug's life:</b></p>
+                <div class="list_content">
+                    <ul class="list">
+                        <li><b>1914</b>- Born in Cresco, Iowa</li><br>
```

Comment:
Kindly use CSS property (margin or padding) to give additional space if required. <br> tag can not be use as descendant of element <li>.